src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -41,7 +41,7 @@ export default class TableViewComponent extends Component {
         const sortKey = this.state.sortKey;
 
         if (!sortKey) {
-            this.setState((state) => ({ ...state, items }));
+            this.setState(() => ({ items }));
```

Comment:
We do not need callback here:
```js
this.setState({ items });
```

Reply:
As far as I know, they are always asynchronous.

Reply:
I'm not so sure